make/openvex.table

```diff
@@ -547,6 +547,34 @@
         ],
         "not_affected": "vulnerable_code_not_present"
       }
+    },
+    {
+      "pkg:github/openssl/openssl@636dfadc70ce26f2473870570bfd9ec352806b1d": "CVE-2016-2183",
+      "status": {
+        "apps": [
+          "pkg:otp/erts@16.0",
+          "pkg:otp/erl_interface@5.6"
+        ],
+        "not_affected": "vulnerable_code_not_present"
+      }
+    }
+  ],
```

Comment:
this has been detected in the `master` branch which has the same openssl version as `maint` (#10224 ).
the reason for not being detected in `maint` is the lack of a backport (#10216 )